### PR TITLE
emag stuff

### PR DESCRIPTION
**Pull request by VernaSaurus**
probably works, i tested it all and it does do things

only adds door, locker, and vendor functionality

### CODE REVIEW

**Comment by VernaSaurus** [comment]
oh no not the closets

**Comment by VernaSaurus** [comment]
fixed the closets

**Comment by PetMudstone** [comment]
Do emags remove access restrictions from vendors?

**Comment by VernaSaurus** [comment]
yes, yes they do

**Comment by PetMudstone** [comment]
This doesn't even have merge conflicts anymore. Come on, github-actions!

File: UnityProject/Assets/Scripts/Items/Emag.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Emag : MonoBehaviour
```
Comment (Bod9001) [inline UnityProject/Assets/Scripts/Items/Emag.cs]: this seems to be unused?

**Comment by corp-0** [comment]
> This doesn't even have merge conflicts anymore. Come on, github-actions!

The action description says it will get rid of the label when there are no conflicts anymore but that seems to be false. We can either disable that action or just manually unlabel when the conflicts are solved.

File: UnityProject/Assets/Scripts/Items/Emag.cs (hunk 1 above)
Comment (VernaSaurus) [reply UnityProject/Assets/Scripts/Items/Emag.cs]: Yeah I completely forgot about this, it's 100% unused

**Comment by VernaSaurus** [comment]
removed the unused file, should be good to merge now

**Pull request merged**